packages/@aws-cdk/aws-ec2-alpha/lib/subnet-v2.ts

```diff
@@ -276,6 +283,135 @@ export class SubnetV2 extends Resource implements ISubnetV2 {
   }
 }
 
+/**
+ * Properties required to import a subnet
+ */
+export interface SubnetV2Attributes {
+  /**
+   * The Availability Zone this subnet is located in
+   *
+   * @default - No AZ information, cannot use AZ selection features
+   */
+  readonly availabilityZone: string;
+
+  /**
+   * The IPv4 CIDR block associated with the subnet
+   *
+   * @default - No CIDR information, cannot use CIDR filter features
+   */
+  readonly ipv4CidrBlock: string;
+
+  /**
+   * The IPv4 CIDR block associated with the subnet
+   *
+   * @default - No CIDR information, cannot use CIDR filter features
+   */
+  readonly ipv6CidrBlock?: string;
+
+  /**
+   * The ID of the route table for this particular subnet
+   *
+   * @default - No route table information, cannot create VPC endpoints
+   */
+  readonly routeTableId?: string;
+
+  /**
+   * The subnetId for this particular subnet
+   */
+  readonly subnetId: string;
+
+  /**
+   * The type of subnet (public or private) that this subnet represents.
+   */
+  readonly subnetType: SubnetType;
+
+  /**
+   * Name of the given subnet
+   *
+   * @default - no subnet name
+   */
+  readonly subnetName?: string;
+
+}
+
+/**
+ * Properties required to import a subnet
+ */
+export interface ImportedSubnetV2Props extends SubnetV2Attributes {}
```

Comment:
what is the value of this interface?

Reply:
if I remember correctly, its to appease the linter which requires that parameter name should be based on the class i.e. `ImportedSubnetV2Props` for class name `ImportedSubnetV2`